sklearn/ensemble/forest.py

```diff
@@ -868,10 +868,11 @@ class RandomForestClassifier(ForestClassifier):
         the generalization accuracy.
 
     n_jobs : int or None, optional (default=None)
-        The number of jobs to run in parallel for both `fit` and `predict`.
-        ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
-        ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
-        for more details.
+        The number of jobs (each tree) to run in parallel for both `fit` 
```

Comment:
Perhaps "The number of trees to build or predict in parallel". I don't find your use of "each" to be clear